pkg/otlp/internal/serializerexporter/consumer.go

```diff
@@ -84,15 +84,21 @@ func (c *serializerConsumer) enrichTags(cardinality string) {
 	const k8sOriginID = ""
 
 	for i := range c.series {
-		tb := tagset.NewHashlessTagsAccumulatorFromSlice(c.series[i].Tags)
```

Comment:
This code looks different on main, you probably need to rebase your topic branch

Reply:
Very much so -- that's a huge undertaking so I'm waiting until further decisions are made before investing the time.